extension-manifest-v3/src/background/reporting.js

```diff
@@ -122,7 +124,7 @@ class Storage {
  */
 class IndexedDBKeyValueStore {
   constructor(dbName, { version = 1, objectStore = 'default' } = {}) {
-    this._dbName = dbName;
+    this._dbName = registerDatabase(dbName);
```

Comment:
Reporting generates db names in module code, so I came up with the idea of storing those names, so that when we want to clean up databases, we keep them.

Reply:
wonder how we can make this work if external dependencies like whotracksme/packages would like to use indexeddb. For now reporting uses dependency injection, but upcoming antitracking has few databases - it may be quite complicated

Reply:
As the last resort, we can initialize: 

```js
const dbs = new Set(['known-database-name-which-cant-be-registered']);
```